src/plugins/featureFlags.ts

```diff
@@ -13,8 +13,8 @@ const defaultFlagSet: LDFlagSet = {
   'hardcoded_regular_wait_time': 0, // by default, use actual wait time
   'hardcoded_priority_wait_time': 0, // by default, use actual wait time
   'banner-text': '', // by default, there is no banner text
-  'advanced-search-enabled': false, // by default don't show advanced search // TODO: remove when feature is ready
-  'staff-payment-enabled': false // by default don't show staff payment // TODO: remove when feature is ready
+  'advanced-search-enabled': false, // by default don't show advanced search // FUTURE: remove when feature is ready
```

Comment:
oh! I think we can remove this and any instances of it now.

Reply:
Isn't "advanced search" what Kial has just implemented? Currently in main branch, this FF is still looked at.

@kialj876 If your new code no longer uses this FF, could you remove it?

Reply:
oh interesting. We had previously used `advanced-search-enabled` for the advanced search feature, maybe it has been re-purposed? 
Good call on following up!

Reply:
No, I was wrong... Advanced Search is the "I don't know my NR number" feature.

I checked LaunchDarkly and this flag is True in all environments, so I will delete it.

Reply:
I deleted 3 obsolete feature flags in our code.

When this feature is deployed, the flags can be deleted from LD.